ios/brave-ios/Sources/BrowserIntentsModels/CustomIntentHandler.swift

```diff
@@ -7,26 +7,44 @@ import CoreSpotlight
 import Intents
 import MobileCoreServices
 
+private func isWebPage(url: URL?) -> Bool {
```

Comment:
There is a `isWebPage` method in Shared's URLExtension class. Should we merge or update them or is it different?

Reply:
This is inside of custom intents. I'd have to import those extensions, which I'm not sure if that's possible for donated intents which run out of process in the Shortcuts app. I'll have to check.

Reply:
okay please check, this is a non-blocking comment